README.md

```diff
@@ -1,29 +1,71 @@
-[![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
+# wallabag - Android App [![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
 
-# wallabag - Android app
+<img src="unnamed.png" align="left"
+width="200"
+    hspace="10" vspace="10">
 
-wallabag app for Android is a companion app for [wallabag](https://www.wallabag.org).
+wallabag is a self hosted read-it-later app.  
+Unlike other services, wallabag is free and open source.  
+wallabag for Android is a companion app for [wallabag](https://www.wallabag.org).  
+wallabag is available on the Google Play Store and F-Droid.
 
-<p align="center">
+<p align="left">
 <a href="https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche">
-  <img alt="Get it on Google Play"
-       src="https://camo.githubusercontent.com/59c5c810fc8363f8488c3a36fc78f89990d13e99/68747470733a2f2f706c61792e676f6f676c652e636f6d2f696e746c2f656e5f75732f6261646765732f696d616765732f67656e657269632f656e5f62616467655f7765625f67656e657269632e706e67" height="80" />
-</a>
+    <img alt="Get it on Google Play"
+        height="80"
+        src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" />
+</a>  
 <a href="https://f-droid.org/app/fr.gaulupeau.apps.InThePoche">
-  <img src="https://camo.githubusercontent.com/f9574a79e3fe61202392c44e55f0bdab261a9561/68747470733a2f2f662d64726f69642e6f72672f62616467652f6765742d69742d6f6e2e706e67"
-       alt="Get it on F-Droid" height="80">
-</a>
-</p>
+    <img alt="Get it on F-Droid"
+        height="80"
+        src="https://f-droid.org/badge/get-it-on.png" />
+ </p>    
+     
+## About
+  
+Wallabag has been made for you to comfortably read and archive your articles.
+You can download wallabag on wallabag.org and follow the instructions to install it on your own server.
+Alternatively, you can directly sign up on wallabag.it or on Framabag.
```

Comment:
I think the first "on" can be better replaced with a "from" and that's what I have done. I don't think "for" would be the right word there.
Also, isn't the second "on" something that should be there for the sentence to make sense?